### DISCUSSION TITLE
NostrGit logo

### DISCUSSION THREAD

**Discussion by cypherhoodlum** [NostrGit/NostrGit#83] [General]
Suggesting this logo for the NostrGit project.

![logo](https://user-images.githubusercontent.com/82510823/223418281-bc13772c-29a7-4c69-a4d3-3cfaef6d7e09.svg)

**Comment by cypherhoodlum** [comment]
Maybe in Bitcoin orange or Nostr purple? brb

**Comment by PeerRich** [comment]
lets keep this for now and just add a text next to it

not a fan making the ostrich into an N


![image](https://user-images.githubusercontent.com/8019099/223419463-e88608ab-d561-4023-addc-597ca192a54c.png)

**Reply by cypherhoodlum** [reply]
To be honest I forgot that already exists. :smiling_face_with_tear: 
In purple would have looked like this (not svg):
![logo-purple](https://user-images.githubusercontent.com/82510823/223425335-f51381a7-826b-4fef-bbdb-3dcc016d73bf.png)
